zcutil/fetch-params.py

```diff
@@ -272,15 +284,16 @@ def create_readme():
 
 def print_intro():
     print('''Zcash - fetch-params.py \n\n''' 
-          '''This script will fetch the Zcash zkSNARK parameters and verify their integrity with sha256sum. \n'''
-          '''If they already exist locally, it will verify SHAs and exit. \n ''')
+          '''This script will fetch the Zcash zkSNARK parameters and verify their  \n'''
+          '''integrity with sha256sum. \n\n'''
```

Comment:
```suggestion
          '''This script will fetch the Zcash zkSNARK parameters and verify their integrity with sha256sum. \n'''
```

Reply:
I don't see the reason for that line break here. It looks weird on my terminal when there is a line break there.

Reply:
Your terminal width is 150 characters, though. I think it's fine to line break here for the benefit of narrower terminals.